man/neptune_init.Rd

```diff
@@ -0,0 +1,73 @@
+\name{neptune_init}
+\alias{neptune_init}
+%- Also NEED an '\alias' for EACH other topic documented here.
+\title{
+Starts a new tracked run, and append it to the top of the Runs table view.
+}
+\description{
+Starts a new tracked run, and append it to the top of the Runs table view. All parameters are optional, hence minimal invocation: run <- neptune_init().
+}
+\usage{
+neptune_init(project = NULL,
```

Comment:
Hmm a lot of params from neptune.init are missing.
custom_run_id, tags, name, description, capture_hardware_metrics, monitoring_namespace, flush_period etc.

Reply:
See https://docs.neptune.ai/api-reference/neptune#.init

Reply:
@needlecapter this one was missed